pkg/ingester/ingester_v2.go

```
@@ -0,0 +1,308 @@
+		return nil, err
+	}
+
+	return &client.LabelNamesResponse{
+		LabelNames: names,
+	}, nil
+}
+
+func (i *Ingester) getTSDB(userID string) *tsdb.DB {
+	i.userStatesMtx.RLock()
+	defer i.userStatesMtx.RUnlock()
+	db, _ := i.TSDBState.dbs[userID]
```

golangci-lint (1.64.8)

S1005: unnecessary assignment to the blank identifier

(gosimple)